grpc/src/js_client/grpc_client.rs

```diff
@@ -0,0 +1,137 @@
+use wasm_bindgen::prelude::*;
+
+use celestia_types::blob::BlobParams;
+use celestia_types::block::Block;
+use celestia_types::state::auth::JsAuthParams;
+use celestia_types::state::auth::JsBaseAccount;
+use celestia_types::state::{JsCoin, TxResponse};
+
+use crate::grpc::GetTxResponse;
+use crate::grpc::TxStatusResponse;
+use crate::grpc::{GasInfo, JsBroadcastMode};
+use crate::Result;
+
+type InnerClient = crate::GrpcClient<tonic_web_wasm_client::Client>;
+
+/// Celestia GRPC client
+#[wasm_bindgen]
+pub struct GrpcClient {
+    client: InnerClient,
+}
+
+#[wasm_bindgen]
+impl GrpcClient {
+    /// Create a new client connected with the given `url`
+    pub async fn new(url: String) -> Result<Self> {
```

Comment:
again just for consistency, as we mostly use `&str` now
```suggestion
    pub async fn new(url: &str) -> Result<Self> {
```